unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ScrollRectSensitivityHandler.cs

```diff
@@ -1,75 +1,75 @@
+        private const float DEFAULT_SENSITIVITY_MULTIPLIER = 1.32f;
 
-    private void SetScrollRectSensitivity() 
-    {
-        myScrollRect.scrollSensitivity = defaultSens * GetScrollMultiplier();
-    }
+        private ScrollRect myScrollRect;
+        private float defaultSens;
 
-    private float GetScrollMultiplier() {
-        switch (GetCurrentOperatingSystem())
+        private void Awake()
         {
-            case OperatingSystemFamily.Windows:
-                return WINDOWS_SENSITIVITY_MULTIPLIER;
-            case OperatingSystemFamily.Linux:
-                return LINUX_SENSITIVITY_MULTIPLIER;
-            case OperatingSystemFamily.MacOSX:
-                return MAC_SENSITIVITY_MULTIPLIER;
-            default:
-                return DEFAULT_SENSITIVITY_MULTIPLIER;
+            myScrollRect = GetComponent<ScrollRect>();
+            defaultSens = myScrollRect.scrollSensitivity;
+            SetScrollRectSensitivity();
         }
-    }
 
-    private OperatingSystemFamily GetCurrentOperatingSystem() {
-#if UNITY_WEBGL
-        return ObtainOsFromWebGLAgent();
-#else
-        return SystemInfo.operatingSystemFamily;
-#endif
-    }
-
-    private OperatingSystemFamily ObtainOsFromWebGLAgent() {
-        String agentInfo = WebGLPlugin.GetUserAgent();
-        if (agentInfo.ToLower().Contains("windows"))
+        private void SetScrollRectSensitivity()
         {
-            return OperatingSystemFamily.Windows;
+            float scrollSensitivity = defaultSens * GetScrollMultiplier();
+            myScrollRect.scrollSensitivity = scrollSensitivity;
         }
-        else if (agentInfo.ToLower().Contains("mac") || agentInfo.ToLower().Contains("osx") || agentInfo.ToLower().Contains("os x"))
+
+        private float GetScrollMultiplier()
         {
-            return OperatingSystemFamily.MacOSX;
+            OperatingSystemFamily currentOperatingSystem = GetCurrentOperatingSystem();
+
+            switch (currentOperatingSystem)
+            {
+                case OperatingSystemFamily.Windows:
+                    return WINDOWS_SENSITIVITY_MULTIPLIER;
+                case OperatingSystemFamily.Linux:
+                    return LINUX_SENSITIVITY_MULTIPLIER;
+                case OperatingSystemFamily.MacOSX:
+                    return MAC_SENSITIVITY_MULTIPLIER;
+                default:
+                    return DEFAULT_SENSITIVITY_MULTIPLIER;
+            }
         }
-        else if (agentInfo.ToLower().Contains("linux"))
-        {
-            return OperatingSystemFamily.Linux;
+
+        private OperatingSystemFamily GetCurrentOperatingSystem() {
+#if UNITY_WEBGL
+            return ObtainOsFromWebGLAgent();
+#else
+            return SystemInfo.operatingSystemFamily;
+#endif
         }
-        else
-        {
+
+        private OperatingSystemFamily ObtainOsFromWebGLAgent() {
+            string agentInfo = WebGLPlugin.GetUserAgent();
+
+            if (agentInfo.ToLower().Contains("windows"))
```

Comment:
Can we improve comparisons a bit to use `StringComparison.OrdinalIgnroeCase` instead of `ToLower()` ?

Reply:
done!